snippets/Google.Datastore.V1Beta3.Snippets/DatastoreDbSnippets.cs

```diff
@@ -660,7 +660,7 @@ public void GroupingQuery()
                 DistinctOn = { "type" },
                 Order =
                 {
-                    { "type", Direction.Ascending },
+                    "type", // If only the name is specified, it's implicitly ascending
                     { "priority", Direction.Ascending }
```

Comment:
I think so :) I would make the second descending, but that wouldn't match the API documentation example.
Maybe I should add a comment saying "// Either direction can be specified explicitly" on the second line?